utilities/class.tapestry-helpers.php

```diff
@@ -447,6 +576,10 @@ public static function checkVideoFileSize($node)
     // https://www.php.net/manual/en/function.filesize.php#113457
     private static function _realFileSize($path)
     {
+        if (!$path) {
+            return PHP_INT_MAX;
```

Comment:
Yes, I included this check because I could think of some cases where the path object would be null (e.g. if `H5P_DEFINED` is false, or if a video node's mediaURL is not a local upload, which was added in 1213). But on second thought, you're right - this doesn't make a lot of sense. All calls to `checkVideoFileSize` are indeed guarded by `videoCanBeUploaded`, which makes these cases impossible. I'll write down a precondition to `checkVideoFileSize` instead.

I think you're also right about the second point. If we really need `_realFileSize` to handle a null path in the future, we might want to let the caller customize the return value when the path is null, since `PHP_INT_MAX` might not be right for all use cases.